### DISCUSSION TITLE
Adding \n to all Args In The Packages Docstrings.

### DISCUSSION THREAD

**Discussion by taylor-ennen** [microsoft/autogen#308] [Ideas]
Before I perform the work or write a script to do so, I'm hoping the maintainers of the project can weigh in on this change.

Overall it is quite simple:

Adding a \n to the end of a Docstring Arg ***vastly*** improves readability given different screen and window sizes, bringing a format into place. This would also be implementable if there is a standard format being implemented before changes can be made, ie. Black format.

I'll attach a couple of screen shots below that illustrate this change and the readability I mentioned above.

## Without the suggested changes:
![image](https://github.com/microsoft/autogen/assets/70374690/c05e7415-8467-4c4c-b1e6-290a3b149eb5)

## With the suggested changes:
![image](https://github.com/microsoft/autogen/assets/70374690/192f04de-d54e-4872-9060-ac3f2f4f2549)

@sonichi I believe this is something I can accomplish on my own if you agree to the proposed changes in formatting standards, I can also help to add to the `formatting tool` used if there is one, I have not looked exhaustively though for a standard formatting tool at this point though.

Side note: I love the project and am working on implementing a Local Inference with a program LM Studio when I noticed these changes could be made, great work on this whole project; to all the contributors!

**Comment by sonichi** [comment]
![image](https://github.com/microsoft/autogen/assets/4250911/f7a1fb1b-7fde-4e8b-a6b5-2a691b04ae6c)
The format looks good to me on https://microsoft.github.io/autogen/docs/reference/agentchat/conversable_agent
How are you viewing it?

**Reply by taylor-ennen** [reply]
The original screenshots I posted were captured while viewing the tooltip in VSCode.

**Reply by sonichi** [reply]
I see. Sure, give it a try. If adding \n makes both tooltip work and doesn't degrade the look on the website, good to make the change.

**Reply by taylor-ennen** [reply]
Awesome. 
May I also ask, is there a formatting tool that is being used in the repo currently? I would be happy to look into how to add a rule to that formatting tool allowing this change to be added automatically to the docstrings.

**Reply by sonichi** [reply]
Yes. They are in https://github.com/microsoft/autogen/blob/main/pyproject.toml
